.github/actions/kots-gke-create/reserve-capacity.yaml

```diff
@@ -22,7 +22,7 @@ metadata:
   name: capacity-res-job
 spec:
   parallelism: 4
-  backoffLimit: 0
+  backoffLimit: 6
```

Comment:
Unrelated to the issue, but should fix flakiness we've been seeing in this test as it will allow this job, which reserves GKE autopilot capacity, to be retried.